src/webgl/material.js

```diff
@@ -476,6 +476,49 @@ p5.prototype.specularMaterial = function(v1, v2, v3, a) {
   return this;
 };
 
+/**
+ * Sets the amount of gloss in the surface of shapes.
+ * Used in combination with specularMaterial() in setting
+ * the material properties of shapes.
+ * @method shininess
+ * @param {Number} shine Degree of Shininess.
+ *                       Defaults to 1.
+ * @chainable
+ * @example
+ * <div>
+ * <code>
+ * function setup() {
+ *   createCanvas(100, 100, WEBGL);
+ * }
+ * function draw() {
+ *   background(0);
+ *   noStroke();
+ *   let locX = mouseX - width / 2;
+ *   let locY = mouseY - height / 2;
+ *   ambientLight(60, 60, 60);
+ *   pointLight(255, 255, 255, locX, locY, 50);
+ *   specularMaterial(250);
+ *   translate(-23, 0, 0);
+ *   shininess(1);
+ *   sphere(20);
```

Comment:
Perhaps worth making these spheres a little more detailed so that the per-vertex lighting doesn't look so odd.

Reply:
@stalgiag Can you please elaborate more on this?

Reply:
Hm I think you can dismiss this, issues with per vertex lighting looking bad in examples is more or less inevitable. This is a separate problem. But just for clarification, I was suggesting that you pass higher than default values into the optional detail parameters of sphere for these examples to make them look better.

Reply:
Oh, I thought you meant something else by "detailed" :sweat_smile: